diagrams.net/assets/stencils/aws/deployment_management.xml

```diff
@@ -253,7 +253,7 @@
 <fillstroke/>
 </foreground>
 </shape>
-<shape name="AWS Elastic BeanStalk Applicaton" h="53.89" w="31.1" aspect="variable" strokewidth="inherit">
```

Comment:
This file is copied from https://github.com/jgraph/drawio/blob/bbe89f62e093c56735281e63ac42b3fccb07425d/src/main/webapp/stencils/aws/deployment_management.xml#L256

You should probably fix this typo there

Reply:
> This file is copied from https://github.com/jgraph/drawio/blob/bbe89f62e093c56735281e63ac42b3fccb07425d/src/main/webapp/stencils/aws/deployment_management.xml#L256
> 
> You should probably fix this typo there

Thanks for pointing that out. Meanwhile I authored this [pull request](https://github.com/jgraph/drawio/pull/3423) that addresses the issue in the upstream repository. I think we leave my change in the kroki repo in place.